docs/configuration/xml.rst

```diff
@@ -26,6 +26,8 @@ Quick Start
 The basic steps to getting configuration set up with your application are:
 
 1. Set up your configuration in JSON or XML files that can be read by ``Microsoft.Extensions.Configuration``.
+  * JSON Configuration uses ``Microsoft.Extensions.Configuration.Json``
+  * XML Configuration uses ``Microsoft.Extensions.Configuration.XML``
```

Comment:
"Configuration" should be lower case - "JSON configuration uses..." and "XML configuration uses..."